src/coreclr/jit/simdcodegenxarch.cpp

```diff
@@ -548,7 +548,7 @@ void CodeGen::genSimd12UpperClear(regNumber tgtReg)
             zroSimd12Elm3 = GetEmitter()->emitSimd16Const(constValue);
         }
 
-        GetEmitter()->emitIns_SIMD_R_R_C(INS_andpd, EA_16BYTE, tgtReg, tgtReg, zroSimd12Elm3, 0);
```

Comment:
`andpd` vs `andps` doesn't matter, but it should've been `andps` since that's "more correct" for `TYP_SIMD12` (e.g. `Vector3` where all fields are `float`).